api/models/settings.py

```diff
@@ -0,0 +1,228 @@
+from api.db import db
+
+
+class ApplicationEnvironament:
```

Comment:
Why to have this class when you can populate some default data and give the user the option from the options, like a drop down.

Reply:
It will give user a option, PLease look at L17, I already added that, so it's more like a radio button